common/flatpak-exports.c

```diff
@@ -383,6 +383,11 @@ flatpak_exports_append_bwrap_args (FlatpakExports *exports,
             }
         }
     }
+  /* As per the os-release specification https://www.freedesktop.org/software/systemd/man/os-release.html
+   * always read-only bind-mount /etc/os-release and /usr/lib/os-release from the host into
+   * the application's /run/host if they exist */
+  else if (g_file_test ("/usr/lib/os-release", G_FILE_TEST_EXISTS))
+    flatpak_bwrap_add_args (bwrap, "--ro-bind", "/usr/lib/os-release", "/run/host/usr/lib/os-release", NULL);
```

Comment:
There's a few places where these could be added, I picked this function so that in case host-etc or host-os are requested the additional bind mounts are avoided. If other functions would be more appropriate let me know and I'll fix it.